regifter.sql

```diff
@@ -16,103 +16,109 @@ CREATE DATABASE regifter;
 -- value - integer
 -- previously_regifted boolean
 
+CREATE TABLE gifts ( id serial primary KEY, giver TEXT, gift TEXT, price INT, previously_regifted BOOLEAN );
 
 -- 
 \echo See details of the table you created
 -- 
-
+\d+ gifts
 
 -- 
 \echo Alter the table so that the column price is changed to value 
 -- 
-
+ALTER TABLE gifts RENAME price TO value;
 
 -- 
 \echo Insert a peach candle, given by 'Santa' thats value is 9 and has been previously regifted
 -- 
-
+INSERT INTO gifts (giver, gift, value, previously_regifted)                                                                                    
+VALUES ('Santa', 'peach candle', 9, TRUE);
 
 --
 \echo Query for all the columns in your gifts table
 -- 
-
+SELECT * FROM gifts;
 
 --
 \echo Uncomment below to insert 5 more gifts
 -- 
 
--- INSERT INTO gifts (gift, giver, value, previously_regifted)
--- VALUES
--- ('peach candle', 'Santa', '9', TRUE),
--- ('cinnamon candle', 'Nick', '19', TRUE),
--- ('soap on a rope', 'Rudolf', '29', FALSE),
--- ('potpurri', 'Elf on the Shelf', '39', TRUE),
--- ('mango candle', 'The Boss', '49', FALSE)
--- ;
+INSERT INTO gifts (gift, giver, value, previously_regifted)
+VALUES
+('Santa', 'peach candle', '9', TRUE),
+('Nick', 'cinnamon candle', '19', TRUE),
+('Rudolf', 'soap on a rope', '29', FALSE),
+('Elf on the Shelf', 'potpurri', '39', TRUE),
+('The Boss', 'mango candle', '49', FALSE)
+;
 
 -- 
 \echo Insert 5 more gifts of your own choosing,  include 1 more candle
 --
-
-
+INSERT INTO gifts (giver, gift, value, previously_regifted)                                                                                    
+VALUES 
+('Santa', 'playstation 5', 500, FALSE),
+('Ash Ketchum', 'pokeballs', 600, FALSE),
+('Santa', 'apple cinnamon wick candle', 20, TRUE),
+('Jovanny', 'URU ore', 1000, TRUE),
+('Santa', 'Skateboard', 0, TRUE);
 
 --
 \echo Query for gifts with a price greater than or equal to 20
 --
-
+SELECT * FROM gifts WHERE value >= 20;
 
 --
 \echo Query for every gift that has the word candle in it, only show the gift column
 --
-
+SELECT * FROM gifts WHERE gift LIKE '%candle%';
 
 --
 \echo Query for every gift whose giver is Santa OR value is greater than 30
 --
-
+SELECT * FROM gifts WHERE giver = 'Santa' AND value >= 30;
 
 --
 \echo Query for every gift whose giver is NOT Santa
 --
-
+SELECT * FROM gifts WHERE giver != 'Santa';
 
 --
 \echo Update the second gift to have a value of 2999
 -- 
-
+UPDATE gifts SET value = 2999 WHERE id = 2;
 
 --
-\echo Query for the updated item
+\echo Query for the updated gift
 --
-
+SELECT * FROM gifts WHERE id = 2 RETURNING *;
```

Comment:
```suggestion
SELECT * FROM gifts WHERE id = 2;
```
`RETURNING *` is redundant here. You're already querying for the entire item by default. I believe it causes a syntax error too!